roles/satellite-clone/tasks/main.yml

```diff
@@ -184,6 +184,13 @@
     state: latest
   when: puppet_version|int == 4
 
+  # Because of https://projects.theforeman.org/issues/30506
+- name: Install Satellite SELinux packages for 6.8 and above
+  yum:
+    name: "{{ selinux_pkgs | join(','') }}"
```

Comment:
Looks like there is an extra quote here `join(','')`, should be `join(',')`

Reply:
Damn VScode trying to be smart lol

Reply:
Fixed